bigbone/src/main/kotlin/social/bigbone/api/entity/Conversation.kt

```diff
@@ -0,0 +1,33 @@
+package social.bigbone.api.entity
+
+import com.google.gson.annotations.SerializedName
+
+/**
+ * Represents a conversation with "direct message" visibility.
+ * @see <a href="https://docs.joinmastodon.org/entities/Conversation/">Mastodon API Conversation</a>
+ */
+data class Conversation(
+    /**
+     * The ID of the conversation in the database.
+     */
+    @SerializedName("id")
+    val id: String? = null,
+
+    /**
+     * Is the conversation currently marked as unread?
+     */
+    @SerializedName("unread")
+    val unread: Boolean? = null,
+
+    /**
+     * Participants in the conversation.
+     */
+    @SerializedName("accounts")
+    val accounts: List<Account>? = null,
```

Comment:
This could have `emptyList()` as default value, to avoid having to check for null.

Reply:
Fixed.